docs/03-pages/01-building-your-application/05-optimizing/10-testing.mdx

```diff
@@ -441,7 +441,9 @@ If you need to add more setup options before each test, it's common to add them
 
 **Optional: Absolute Imports and Module Path Aliases**
 
-If your project is using [Module Path Aliases](/docs/pages/building-your-application/configuring/absolute-imports-and-module-aliases), you will need to configure Jest to resolve the imports by matching the paths option in the `jsconfig.json` file with the `moduleNameMapper` option in the `jest.config.js` file. For example:
+The path configuration, be it the base URL or an alias, must match in both `tsconfig.json` and `jest.config.js` so that Jest can resolve the imports.
```

Comment:
```suggestion
The path configuration (base URL or an alias) must match in both `tsconfig.json` and `jest.config.js` so that Jest can resolve the imports.
```